test-requirements.txt

```diff
@@ -77,8 +77,9 @@ click==8.1.3 \
     --hash=sha256:7682dc8afb30297001674575ea00d1814d808d6a36af415a82bd481d37ba7b8e \
     --hash=sha256:bb4d8133cb15a609f44e8213d9b391b0809795062913b383c62be0ee95b1db48
     # via black
-cloudimg==1.1.0 \
-    --hash=sha256:a1a70e04710f382a8c5185afeef3cbd63319c35e0c793ecc5c6b7bb0c6517bd5
+cloudimg>=1.2.0 \
```

Comment:
The >= is meant to go where I commented on requirements.in and not here.  You don't want to try modifying these files manually, they are generated by pip-compile.

Here's the process I'd suggest when updating minimum version of a dependency:

1. Set cloudimg >= 1.2.0 in requirements.in
2. Run `tox -e pip-compile`, this will probably make several changes to the .txt files
3. Run `git add -p` to walk through the changes to the .txt files and add only the changes relating to cloudimg